lib/engine/g_1856/corporation.rb

```diff
@@ -36,10 +36,6 @@ def can_buy?
         false
       end
 
-      def can_par?
-        self == @game.national ? false : super
-      end
-
       def par!
```

Comment:
don't need to do it for this PR, but this should be able to allow you to delete the game specific corp